openedx_lti_tool_plugin/views.py

```diff
@@ -264,7 +337,33 @@ def get(
         """
         return self.render_to_response(
             {
-                'lms_root_url': getattr(settings, 'LMS_ROOT_URL', ''),
                 'xblock_url': reverse(f'{AppConfig.name}:lti-xblock', args=[unit_key]),
             },
         )
+
+
+@method_decorator([xframe_options_exempt, login_required], name='dispatch')
```

Comment:
Is it necessary to disable x-frame-options? @kuipumu

Reply:
@kuipumu Yes, otherwise the launch will not work if it's inside an iframe, like in the first screenshot of the PR description.